cfme/tests/configure/test_visuals.py

```diff
@@ -43,6 +45,12 @@ def set_list():
     visual.list_view_limit = 20
 
 
+@pytest.yield_fixture(scope="module")
+def set_default_page():
```

Comment:
As this isn't **yielding** anything, it is probably best here to do this as a finalizer function instead.

```python
def set_default_page():
    visual.set_login_page = "Cloud Intelligence / Dashboard"

def test_start_page(request.....
    request.addfinalizer(set_default_page)
```